presto-main/src/main/java/com/facebook/presto/sql/planner/iterative/rule/AddIntermediateAggregations.java

```diff
@@ -113,11 +113,17 @@ public Result apply(AggregationNode aggregation, Captures captures, Context cont
         PlanNode source = rewrittenSource.get();
 
         if (getTaskConcurrency(session) > 1) {
+            Map<VariableReferenceExpression, Aggregation> inputsAsOutputs = inputsAsOutputs(aggregation.getAggregations(), types);
```

Comment:
Can we name the variable differently from the function name? This is quite confusing.

Reply:
Yeah - I had no idea what the function does lol. But I will rename the variable.

Reply:
> Yeah - I had no idea what the function does lol. But I will rename the variable.

done